beep/structure.py

```diff
@@ -749,7 +749,8 @@ def from_maccor_file(cls, filename, include_eis=True, validate=False):
         return cls(data, metadata, eis, validate, filename=filename)
 
     def determine_structuring_parameters(self, v_range=None, resolution=1000,
-                                         nominal_capacity=1.1, full_fast_charge=0.8):
+                                         nominal_capacity=1.1, full_fast_charge=0.8,
+                                         parameters_path='data-share/raw/parameters'):
```

Comment:
Could we also add the argument to the docstring?

Reply:
Added arg to docstring